docs/_apps/swt-detection/v4.0/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: posts
+classes: wide
+title: "Scenes-with-text Detection (v4.0)"
+date: 2024-02-27T21:53:26+00:00
+---
+## About this version
+
+* Submitter: [marcverhagen](https://github.com/marcverhagen)
+* Submission Time: 2024-02-27T21:53:26+00:00
+* Prebuilt Container Image: [ghcr.io/clamsproject/app-swt-detection:v4.0](https://github.com/clamsproject/app-swt-detection/pkgs/container/app-swt-detection/v4.0)
+* Release Notes
+
+    > Version 4.0
```

Comment:
This version contains a pretty major re-organization of the output format. To that end I think it could've been better with a more detailed release note (git tag annotation). But I'm releasing new SDK versions that reflex the output format used in this version sometime soon, so I think we can use the new SDK release as a chance to give a minor bump-up to the SWT app and its release note then.

Reply:
Agreed. We also need to streamline where all the release information is defined. Unless I am mistaken, we have (1) a README file with space to explain what this version does, (2) a special pull request which details all the changes, and (3) a comment in the commit tag. There is too much redundancy there.